feat(llm): rename category (#3161)
scaleway/docs-content: ai-data/managed-inference/index.mdx

        icon="rocket"

>         description="Learn how to create, connect to, and delete an LLM endpoint in a few steps."

Reviewer: ```suggestion
        description="Learn how to create, connect to, and delete a Managed Inference endpoint in a few steps."
```

        label="View Quickstart"
        url="/ai-data/managed-inference/quickstart/"
    />
    <SummaryCard
        title="Concepts"
        icon="information-circle-outline"
        description="Core concepts that give you a better understanding of Scaleway Managed Inference."
        label="View Concepts"
        url="/ai-data/managed-inference/concepts/"
    />
    <SummaryCard
        title="How-tos"
        icon="help-circle-outline"

>         description="Check our guides about creating and managing LLM endpoints."

Reviewer: ```suggestion
        description="Check our guides about creating and managing Managed Inference endpoints."
```

Reviewer: Not sure about these

        label="View How-tos"
        url="/ai-data/managed-inference/how-to/"
    />
    <SummaryCard
        title="Additional content"
        icon="book-open-outline"

>         description="Guides to help you choose an LLM endpoint, understand pricing and advanced configuration."

Reviewer: ```suggestion
        description="Guides to help you choose a Managed Inference endpoint, understand pricing and advanced configuration."
```

        label="View additional content"
        url="/ai-data/managed-inference/reference-content/"
    />
</Grid>

<ClickableBanner
    productLogo="cli"
    title="Managed Inference API"

>     description="Learn how to create and manage your Scaleway LLM endpoints through the API."

Reviewer: ```suggestion
    description="Learn how to create and manage your Scaleway Managed Inference endpoints through the API."
```
